README.md

```diff
@@ -49,6 +49,29 @@ profile_metrics_alerting::tools::whitelabel_subtitle: "Managed by ORGANIZATION G
 profile_metrics_alerting::tools::whitelabel_title: "Monitoring and Telemetry Interface"
 ```
 
+To pause/start alerts on a schedule (applicable for old style alerts only), you can use these parameters:
+```yaml
+# Format for command is similar to running alert_toggle.sh by hand, see README in files/root/grafana_tools
+profile_metrics_alerting::alert_cycle::crons:
+   "Pause mforgehn1 alerts":
+    command: "pause tag \"mforgehn1\""
+    hour: 21
+    minute: 55
+    weekday: 0
+  "Start mforgehn1 alerts":
+    command: "start tag \"mforgehn1\""
+    hour: 0
+    minute: 45
+    weekday: 1
+profile_metrics_alerting::alert_cycle::cycle_alerts: true
+```
+You'll want to define your crons for `profile_metrics_alerting::alert_cycle::crons` in any role that uses this repo (For example if you have a primary/secondary role). This is because if you swap roles you want the alert pausing/starting crons to be removed from the secondary role so you don't get alerts enabled in multiple places leading to duplicate alerts. 
```

Comment:
This isn't obvious to me how this works for secondary servers. It seems like the secondary roles should never `start` any alerts.
Perhaps I'm misinterpreting what this line is saying.

Reply:
I was trying to make the alert cycling properly handle someone swapping between a node between the metrics_primary and metrics_secondary roles.

If a node goes from primary to secondary, we want the crons that cycle the alerts to go away (or at least disable themselves). Since crons that aren't managed by puppet do not get purged, we need a way to tell Puppet the name of each cron we want to purge. So my comment here is talking about how you want to define all your crons in both the metrics_primary and metrics_secondary role, like how I did here : https://git.ncsa.illinois.edu/ici/asd/asd-pup-control/-/compare/main...SVC-5148%2Fcycle_alerts

It's ugly and I don't really like doing it that way, but I didn't see any immediate better ways to handle it. Tested seeing if I could add a standard prefix to all the crons, and tell puppet to delete all crons that start with 'alert_cycle_*', but it doesn't look like I can give wildcards in the resource title like that since I got errors.

I'll add some clarification to the README to better explain it.

Reply:
The current status of alerting on the `metrics_secondary` servers is kind of messy and not obvious how they work.
But currently if the `profile_metrics_alerting::tools::backup_and_sync_enable` is `TRUE` (which it is on `metrics_primary`) then that script does a `./alert_toggle.sh pause tag all` as a last step on the server that it syncs to (the destination).

Currently this has the result of pausing all alerts on `metrics02`. But now the changes in this branch are now unpausing alerts for some specific hosts on metrics02 (at least until the next sync gets pushed to it).

I do not think we ever want any alerts enabled on `metrics02` unless `metrics01` fails and we need to temporarily make `metrics02` take the `metrics` CNAME and switch to the primary role.

But I can see that perhaps we do want to enable and test alerts from `metrics-test`, which unfortunately also uses the `metrics_secondary` role.

Perhaps we really need to switch up these roles more so that they are more clear what they are doing. Until now, the `metrics_primary` role primarily had the following distinctions about it:
- use `metrics.ncsa.illinois.edu` for its web configuration
- enable `profile_metrics_alerting::tools::backup_and_sync_enable` to sync data to `metrics02`

Reply:
> But now the changes in this branch are now unpausing alerts for some specific hosts on metrics02 (at least until the next sync gets pushed to it)

This part isn't a problem, the crons for pausing/un-pausing never set foot on metrics-test or metrics02. By default `profile_metrics_alerting::alert_cycle::enable_cycle_alerts` is set to false, and the only place I set that to true is in `data/role/metrics_primary.yaml`. The annoying and confusing part is that I do still define the crons themselves over in `data/role/metrics_secondary.yaml`, but that's only so that I can give Puppet a list of crons that it should delete if they happen to exist. Which they would exist if a node ever changed from role `metrics_primary` to `metrics_secondary`.

If we ever wanted to do some testing where metrics-test would have the crons to cycle alerts on, we could keep it in the same `metrics_secondary` role and then set `profile_metrics_alerting::alert_cycle::enable_cycle_alerts: true` in `data/nodes/metrics-test.ncsa.illinois.edu.yaml`